index.bs

```diff
@@ -896,15 +906,100 @@ spec: mimesniff; urlPrefix: https://mimesniff.spec.whatwg.org/#
         </p>
       </section>
 
-      <section>
-        <h4 id='is-encrypted-decode-supported'>
-          <dfn>Check Encrypted Decoding Support</dfn>
-        </h4>
+      <section algorithm>
+        <h4 id="check-mime-type-support"><dfn abstract-op noexport lt="Check MIME Type Support|check MIME type support">Check MIME Type Support</dfn></h4>
+        <p>
+          To [$check MIME type support$], given a [=MIME type record=] |mimeType|,
+          a string |media|, and either a {{MediaEncodingType}} or
+          {{MediaDecodingType}} |encodingOrDecodingType|, perform the following
+          steps. They return <code>supported</code> if
+          the MIME type is [=supported by the user agent=], <code>unsupported</code>
+          if the MIME type is not supported:
+        </p>
+        <ol>
+          <li>
+            If |encodingOrDecodingType| is {{MediaEncodingType/webrtc}} ({{MediaEncodingType}})
+            or {{MediaDecodingType/webrtc}} ({{MediaDecodingType}}):
+            <ol>
+              <li>
+                <p>
+                  If |mimeType| is not one that is used with RTP, as defined in the
+                  specifications of the corresponding RTP payload formats
+                  [[IANA-MEDIA-TYPES]] [[RFC6838]], return <code>unsupported</code>.
+                </p>
+                <p class="note">
+                  The codec name is typically specified as subtype and zero or more
+                  parameters may be present depending on the codec.
+                </p>
+              </li>
+              <li>
+                If |mimeType| is not [=supported by the user agent=], return
+                <code>unsupported</code>.
+              </li>
+              <li>
+                Return <code>supported</code>.
+              </li>
+            </ol>
+          </li>
+          <li>
+            If |media| is <code>audio</code> and |mimeType|'s [=MIME type/type=]
+            is neither <code>audio</code> nor <code>application</code>,
+            return <code>unsupported</code>.
+          </li>
+          <li>
+            If |media| is <code>video</code> and |mimeType|'s [=MIME type/type=]
+            is neither <code>video</code> nor <code>application</code>,
+            return <code>unsupported</code>.
+          </li>
+          <li>
+            If the combined <code>type</code> and <code>subtype</code> members
+            of |mimeType| imply a single media codec:
```

Comment:
"imply a single media codec" is vague. Making it more precise would involve listing the specific MIME types where a codecs parameter isn't also needed. Related: #235

Reply:
I dropped an issue reference in this PR at the most appropriate spot I could find.